.github/workflows/build.yml

```diff
@@ -10,21 +10,18 @@ on:
 permissions:
   contents: read
 
-env:
-  NODE_VERSION: 16
-
 jobs:
   build-tsc:
     runs-on: ubuntu-latest
     timeout-minutes: 10
     if: github.event.pull_request.draft == false
 
-    name: Build for node ${{ env.NODE_VERSION }}
```

Comment:
according to docs `env.XY` cannot be used in name
https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability